package.json

```diff
@@ -163,7 +163,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-no-only-tests": "^2.6.0",
-    "ethereum-waffle": "^3.4.4",
```

Comment:
`ethereum-waffle@3.4.4` depends on deprecated `@ensdomains/ens`. Although it is not runtime dependency it provides a bit more confidency in tests